src/components/item-details/title.tsx

```diff
@@ -6,12 +6,13 @@ type Props = {
   title: string;
   likeCount?: number;
   itemId?: number;
+  tokenId?: number;
 };
 
-const ProductTitle = ({ className, title, likeCount = 0, itemId }: Props) => (
+const ProductTitle = ({ className, title, tokenId, likeCount = 0, itemId }: Props) => (
   <div className={clsx('pd-title-area', className)}>
     <h4 className="title">
-      {title}#{itemId}
+      {title}#{tokenId}
```

Comment:
Why do you show the tokenId here and not the itemId? In the future, if we list NFTs from different collections, different NFTs might have the same tokenId. After listing, the ItemID should be unique.

Reply:
I did it because i thought that the tokenId was unique and a kind of id but just for the NFT and not for the database.
Like an ID to be shown to the user.
As you tell me that the itemId will be used in the future, I'll change it